Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -49,3 +49,40 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### Spec
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
+
+
+### Consuming StorageClass
+
+Once the RadosNamespace is created, a rbd-based StorageClass can be created to
+consume exactly this Namespace. For this purpose, the  `ClusterID value of the
+Radosnamespace needs to be put into the `ClusterID` field of the StorageClass
+spec  which usually holds the kubernetes namespace  hosting the rook-ceph
+cluster.
+
+## Extracting the ClusterID
+
+ ```shell
+ $ kubectl -n rook-ceph get cephblockpoolradosnamespace -o yaml
+
+  status:
+    info:
+      clusterID: 80fc4f4bacc064be641633e6ed25ba7e
+```
+
+## Defining the StorageClass
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-ceph-block
+provisioner: rook-ceph.rbd.csi.ceph.com # csi-provisioner-name
+parameters:
+  clusterID: 80fc4f4bacc064be641633e6ed25ba7e
+```
+
```

Comment:
nit: remove blank lines at end of file

Reply:
markdownlint should be catching this. AFAICT, this PR is based on an out of date fork that doesn't have the markdownlint github action configured. @obnoxxx please pull the latest Rook upstream changes and rebase on top.

Reply:
Actually, looking more deeply, I think the fork is up to date, but markdownlint isn't running as part of the doc CI. I need to look into why.

Reply:
Got it! https://github.com/rook/rook/pull/14178